### ISSUE TITLE
Add a way to mat accordion to have a title row before the expansion panes

### ISSUE THREAD

**Issue by tbnovaes** [angular/material2#15342]
#### Please describe the feature you would like to request.

It would be great to be able to have it. Since sometimes we want to add accordions to a specific section, and even tho accomplish it with a mat-card and some css is possible, it requires a lot of work if we need to do it in many places.

Having `mat-accordion-header` and `mat-accordion-title` would be nice.

<img width="1109" alt="screen shot 2019-02-28 at 2 02 03 pm" src="https://user-images.githubusercontent.com/5142733/53591279-a05fa500-3b61-11e9-896a-5cc1160b374d.png">

#### What is the use-case or motivation for this proposal?
Want to divide the accordions into sections, which is a really common use case.

**Comment by Sriharish** [comment]
Have you tried using a disabled `<mat-expansion-panel>`? I simply pass an `<h3>` into its `<mat-expansion-panel-header>` and that's it! Here have a [look at this sample](https://stackblitz.com/edit/angular-material2-issue-71uquo?file=app%2Fapp.component.html). You may still need a little CSS if you want to be able to adjust the margins between panels, but that might be a different issue.

**Comment by tbnovaes** [comment]
@Sriharish yes that's a nice workaround, and in fact far easier than use a card for that. Thanks!

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
